coordinator/src/routes/health/get-metrics.ts

```diff
@@ -0,0 +1,26 @@
+import Hapi from "@hapi/hapi"
+import * as osu from "node-os-utils"
+
+export default [
+  {
+    method: "GET",
+    path: "/_status",
```

Comment:
should this have a different path assuming that the status route still exists?

Reply:
Yep, updated